src/platform/plugins/shared/dashboard/public/dashboard_listing/get_dashboard_listing_tabs.tsx

```diff
@@ -0,0 +1,186 @@
+    goToDashboard,
+    getDashboardUrl,
+    useSessionStorageIntegration,
+    initialFilter,
+    contentTypeFilter: TAB_IDS.DASHBOARDS,
+  });
+
+  const dashboardFavoritesClient = useMemo(() => {
+    return new FavoritesClient(DASHBOARD_APP_ID, DASHBOARD_SAVED_OBJECT_TYPE, {
+      http: coreServices.http,
+      usageCollection: usageCollectionService,
+      userProfile: coreServices.userProfile,
+    });
+  }, []);
+
+  return (
+    <TableListViewKibanaProvider
+      {...getBaseKibanaProviderProps()}
+      favorites={dashboardFavoritesClient}
+      contentInsightsClient={contentInsightsClient}
+    >
+      <DashboardUnsavedListing
+        goToDashboard={goToDashboard}
+        unsavedDashboardIds={unsavedDashboardIds}
+        refreshUnsavedDashboards={refreshUnsavedDashboards}
+      />
+      <TableListViewTable<DashboardListingUserContent>
+        tableCaption={tableListViewTableProps.title}
+        {...tableListViewTableProps}
+        {...parentProps}
+      />
+    </TableListViewKibanaProvider>
+  );
+};
+
+const VisualizationsTabContent = ({
+  goToDashboard,
+  getDashboardUrl,
+  useSessionStorageIntegration,
+  initialFilter,
+  parentProps,
+}: TabContentProps) => {
+  const { euiTheme } = useEuiTheme();
+  const { tableListViewTableProps } = useDashboardListingTable({
+    goToDashboard,
+    getDashboardUrl,
+    useSessionStorageIntegration,
+    initialFilter,
+    contentTypeFilter: TAB_IDS.VISUALIZATIONS,
+  });
+
+  return (
+    <TableListViewKibanaProvider {...getBaseKibanaProviderProps()}>
+      <div
+        css={css`
+          .visListingTable__typeImage,
+          .visListingTable__typeIcon {
+            margin-right: ${euiTheme.size.s};
+            position: relative;
+            top: -1px;
+          }
+
+          .visListingTable__typeImage {
+            ${logicalSizeCSS(euiTheme.size.base, euiTheme.size.base)};
+          }
+        `}
+      >
+        <TableListViewTable<DashboardListingUserContent>
+          tableCaption={tableListViewTableProps.title}
+          {...tableListViewTableProps}
+          {...parentProps}
+        />
+      </div>
+    </TableListViewKibanaProvider>
+  );
+};
+
+export const getDashboardListingTabs = ({
+  goToDashboard,
+  getDashboardUrl,
+  useSessionStorageIntegration,
+  initialFilter,
+  listingViewRegistry,
+}: GetDashboardListingTabsParams): TableListTab<DashboardListingUserContent>[] => {
+  const commonProps = {
+    goToDashboard,
+    getDashboardUrl,
+    useSessionStorageIntegration,
+    initialFilter,
+  };
+
+  const dashboardsTab: TableListTab<DashboardListingUserContent> = {
+    title: i18n.translate('dashboard.listing.tabs.dashboards.title', {
+      defaultMessage: 'Dashboards',
+    }),
+    id: TAB_IDS.DASHBOARDS,
+    getTableList: (parentProps) => (
+      <DashboardsTabContent {...commonProps} parentProps={parentProps} />
+    ),
+  };
+
+  const visualizationsTab: TableListTab<DashboardListingUserContent> = {
```

Comment:
Why does visualizations tab not use `listingViewRegistry`? It seems like some visualization code is bleeding into Dashboards because of this. For example, its seems like a code smell to need to define `getReadOnlyVisualizationMessage` in dashboard plugin. Could visualizations tab be moved to `listingViewRegistry` and then remove all visualizations specific changes from dashboard plugin.